philippa-choong/week5/wed/plants_v2/app/models/plant.rb

```diff
@@ -0,0 +1,3 @@
+class Plant < ApplicationRecord
+    belongs_to :buyer
```

Comment:
you need to add `optional: true`, since you don't have `buyer_id` in the form or controller. So it cannot create an item without the buyer its suppose to belong to